api/release_notes/2019-release-notes.md

```diff
@@ -4,6 +4,24 @@ title: 2019 Release Notes
 
 # 2019 Reactor Release Notes
 
+## 2019-09-04
+
+### 1) Technical Accounts can use Product Profiles
+
+When we first released the Launch API, Adobe I/O integrations were not fully integrated into Adobe's Admin Console where regular user accounts are controlled.  As a result, these integrations had a different set of available permissions.
+
+Adobe I/O integrations can now use the same permission groups (called Product Profiles) - created and managed in Adobe's [Admin Console](https://adminconsole.adobe.com/) - that regular users use.
+
+When you login to the Adobe I/O console to create a new integration (or update an existing integration), instead of the older role selector, you'll now see a list of product profiles from the Admin Console.  You'll need to select the product profile you want the integration to belong to.
```

Comment:
They've been able to select profiles for a while. I would drop the `now` to avoid confusion.

Reply:
Also, you can select multiple profiles

Reply:
Fixed these.